src/deployment-guides/tko-on-vsphere-with-tanzu-deploy.md

```diff
@@ -0,0 +1,1036 @@
+| TKG Management Network     | TKG-Management  | 1681 | 172.16.81.1/27 | Yes          | No                                  |
+| TKG Workload Network01     | TKG-Workload    | 1682 | 172.16.82.1/24 | Yes          | No                                  |
+| TKG VIP Network            | TKG-Cluster-VIP | 1683 | 172.16.83.1/24 | No           | 172.16.83.101 - 172.16.83.250|
+
+After you have created the required networks, the network section in your vSphere environment must have the port groups as shown in the following screen capture:
+
+![](./img/tko-vsphere-with-tanzu/image54.jpg)
+
+### <a id=firewall-requirements> </a> Firewall Requirements
+
+Ensure that the firewall is set up as described in [Firewall Recommendations](../reference-designs/tko-on-vsphere-with-tanzu.md#firewall-recommendations).
+
+
+### <a id=resource-pools> </a>Resource Pools
+
+Ensure that resource pools and folders are created in vCenter. The following table shows a sample entry for the resource pool and folder. Customize the resource pool and folder name for your environment.
+
+| Resource Type      | Resource Pool name | Sample Folder name |
+| ------------------ | ------------------ | ------------------ |
+| NSX ALB Components | NSX-ALB            | NSX-ALB-VMS        |
+
+## Deployment Overview
+
+The following are the high-level steps for deploying Tanzu Kubernetes operations on vSphere networking backed by VDS:
+
+1.  [Deploy and Configure NSX Advanced Load Balancer](#config-nsxalb)
+2.  [Deploy Tanzu Kubernetes Grid Supervisor Cluster](#deployTKGS)
+3.  [Create and Configure vSphere Namespaces](#create-namespace)
+4.  [Deploy Tanzu Kubernetes Clusters (Workload Clusters)](#deploy-workload-cluster)
+5.  [Integrate Tanzu Kubernetes Clusters with SaaS Endpoints](#integrate-saas)
+6.  [Deploy User-Managed Packages on Tanzu Kubernetes Grid Clusters](#deploy-user-managed-packages)
+
+## <a id=config-nsxalb> </a> Deploy and Configure NSX Advanced Load Balancer
+
+NSX Advanced Load Balancer is an enterprise-grade integrated load balancer that provides L4- L7 load balancer support. We recommended deploying NSX Advanced Load Balancer for vSphere deployments without NSX-T, or when there are unique scaling requirements.
+
+NSX Advanced Load Balancer is deployed in write access mode in the vSphere environment. This mode grants NSX Advanced Load Balancer Controllers full write access to the vCenter. Full write access allows automatically creating, modifying, and removing Service Engines and other resources as needed to adapt to changing traffic needs.
+
+For a production-grade deployment, we recommend deploying three instances of the NSX Advanced Load Balancer Controller for high availability and resiliency.
+
+The following table provides a sample IP address and FQDN set for the NSX Advanced Load Balancer controllers:
+
+| Controller Node    | IP Address   | FQDN                   |
```

Comment:
To address spelling issues here, please wrap this table with cSpell:disable/enable syntax.

Reply:
added.